diff --git a/nvh/bitarray.hpp b/nvh/bitarray.hpp
index 5a762b3..f1354ff 100644
--- a/nvh/bitarray.hpp
+++ b/nvh/bitarray.hpp
@@ -21,8 +21,10 @@
 #ifndef NV_BITARRAY_H__
 #define NV_BITARRAY_H__
 
+#include <nvp/NvFoundation.h>
 #include <algorithm>
 #include <platform.h>
+#include <cstdint>
 #if(defined(NV_X86) || defined(NV_X64)) && defined(_MSC_VER)
 #include <intrin.h>
 #endif
diff --git a/nvh/nvprint.cpp b/nvh/nvprint.cpp
index bdbe186..c53d1a4 100644
--- a/nvh/nvprint.cpp
+++ b/nvh/nvprint.cpp
@@ -22,6 +22,7 @@
 
 #include <mutex>
 #include <vector>
+#include <cstring>
 
 #ifdef _WIN32
 #include <windows.h>
diff --git a/nvh/nvprint.hpp b/nvh/nvprint.hpp
index ee14dc0..febd9d0 100644
--- a/nvh/nvprint.hpp
+++ b/nvh/nvprint.hpp
@@ -27,6 +27,7 @@
 #include <stdarg.h>
 #include <stdio.h>
 #include <stdlib.h>
+#include <cstdint>
 
 
 /**
diff --git a/nvh/parametertools.cpp b/nvh/parametertools.cpp
index 58bf942..f70ac6c 100644
--- a/nvh/parametertools.cpp
+++ b/nvh/parametertools.cpp
@@ -21,6 +21,7 @@
 #include "parametertools.hpp"
 #include "nvprint.hpp"
 
+#include <cstring>
 #include <algorithm>
 
 namespace nvh {
diff --git a/nvh/parametertools.hpp b/nvh/parametertools.hpp
index f44bfcc..7e01eef 100644
--- a/nvh/parametertools.hpp
+++ b/nvh/parametertools.hpp
@@ -24,6 +24,7 @@
 
 #include "platform.h"
 
+#include <cfloat>
 #include <climits>
 #include <functional>
 #include <string>
diff --git a/nvh/trangeallocator.hpp b/nvh/trangeallocator.hpp
index d1e9822..f2cb453 100644
--- a/nvh/trangeallocator.hpp
+++ b/nvh/trangeallocator.hpp
@@ -25,7 +25,7 @@
 #include <stdint.h>
 #include <stdio.h>
 
-#include <NvFoundation.h>  // for NV_X86 and NV_X64
+#include "nvp/NvFoundation.h"  // for NV_X86 and NV_X64
 
 #if(defined(NV_X86) || defined(NV_X64)) && defined(_MSC_VER)
 #include <intrin.h>
diff --git a/nvp/perproject_globals.cpp b/nvp/perproject_globals.cpp
index fc74313..fcc9ccb 100644
--- a/nvp/perproject_globals.cpp
+++ b/nvp/perproject_globals.cpp
@@ -21,7 +21,7 @@
 
 std::string getProjectName()
 {
-  return PROJECT_NAME;
+  return "RT64VK";  //return PROJECT_NAME;
 }
 
 bool isAftermathAvailable()
diff --git a/nvvk/commands_vk.cpp b/nvvk/commands_vk.cpp
index 8207a08..44660f6 100644
--- a/nvvk/commands_vk.cpp
+++ b/nvvk/commands_vk.cpp
@@ -21,7 +21,7 @@
 #include <algorithm>
 #include <platform.h>
 
-
+#include <nvp/NvFoundation.h>
 #include "commands_vk.hpp"
 #include "error_vk.hpp"
 
